packages/dapp/src/utils/3box.js

```diff
@@ -9,29 +7,30 @@ export const getProfile = async account => {
     imageHash: '',
     imageUrl: `https://avatars.dicebear.com/api/jdenticon/${address}.svg`,
   };
-  const origin = window.location.origin;
-  const fetchOptions = {
-    mode: origin.startsWith('http://localhost') ? 'no-cors' : 'cors',
-  };
+  // const origin = window.location.origin;
+  // const fetchOptions = {
+  //   mode: origin.startsWith('http://localhost') ? 'no-cors' : 'cors',
+  // };
 
-  const response = await fetch(
-    `${BOX_ENDPOINT}/profile?address=${encodeURIComponent(address)}`,
-    fetchOptions,
-  );
-  if (response.ok && response.status === 200) {
-    const boxProfile = await response.json();
-    const imageHash =
-      boxProfile &&
-      boxProfile.image &&
-      boxProfile.image[0] &&
-      boxProfile.image[0].contentUrl &&
-      boxProfile.image[0].contentUrl['/'];
-    if (imageHash) {
-      profile.imageHash = imageHash;
-      profile.imageUrl = `${IPFS_ENDPOINT}/ipfs/${imageHash}`;
-    }
-    profile.name = boxProfile.name;
-    profile.emoji = boxProfile.emoji;
-  }
+  // 3Box service no longer works as of 07/10/2023
+  // const response = await fetch(
+  //   `${BOX_ENDPOINT}/profile?address=${encodeURIComponent(address)}`,
+  //   fetchOptions,
+  // );
+  // if (response.ok && response.status === 200) {
+  //   const boxProfile = await response.json();
+  //   const imageHash =
+  //     boxProfile &&
+  //     boxProfile.image &&
+  //     boxProfile.image[0] &&
+  //     boxProfile.image[0].contentUrl &&
+  //     boxProfile.image[0].contentUrl['/'];
+  //   if (imageHash) {
+  //     profile.imageHash = imageHash;
+  //     profile.imageUrl = `${IPFS_ENDPOINT}/ipfs/${imageHash}`;
+  //   }
+  //   profile.name = boxProfile.name;
+  //   profile.emoji = boxProfile.emoji;
+  // }
```

Comment:
Personally I'm not a fan of commenting out code. It generally doesn't get cleaned up and sits around for too long. I know I said removing 3box is not necessarily in scope, but simply commenting it out is also not ideal.

Reply:
yeah, I generally agree about not being a fan of commenting out code. My thought was to leave this in in case the 3box service came back form the dead. But I will remove it if you think that is better.